src/LegacyCallHandler.tsx

```diff
@@ -194,13 +200,14 @@ export default class LegacyCallHandler extends EventEmitter {
     }
 
     public unSilenceCall(callId: string): void {
+        if (this.isForcedSilent) return;
```

Comment:
This seems like it'll make the `Sound on` button unresponsive without an obvious reason